hs_core/management/commands/repair_resource.py

```diff
@@ -67,13 +71,18 @@ def handle(self, *args, **options):
             print("FILTERING TO INCLUDE PUBLISHED RESOURCES ONLY")
             resources = resources.filter(raccess__published=True)
 
-        if days:
-            print(f"FILTERING TO INCLUDE RESOURCES UPDATED IN LAST {days} DAYS")
+        if updated_since:
+            print(f"FILTERING TO INCLUDE RESOURCES UPDATED IN LAST {updated_since} DAYS")
             if resources_ids:
-                print("Your supplied resource_ids will be filtered by the --days that you provided. ")
-            cuttoff_time = timezone.now() - timedelta(days)
+                print("Your supplied resource_ids will be filtered by the --updated_since days that you provided. ")
+            cuttoff_time = timezone.now() - timedelta(updated_since)
             resources = resources.filter(updated__gte=cuttoff_time)
 
+        if ignore_repaired_since:
+            print(f"FILTERING TO INCLUDE RESOURCES NOT REPAIRED IN THE LAST {ignore_repaired_since} DAYS")
+            cuttoff_time = timezone.now() - timedelta(days=ignore_repaired_since)
```

Comment:
For consistency may be change line 78 to pass the `days` to `timedelta `call?

Reply:
added "days" to the timedelta function...I think that's what you meant...LMK if I missunderstood thanks!